src/Symfony/Component/Notifier/Bridge/Ntfy/NtfyTransport.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\Ntfy;
+
+use Symfony\Component\Notifier\Exception\LogicException;
+use Symfony\Component\Notifier\Exception\TransportException;
+use Symfony\Component\Notifier\Exception\UnsupportedMessageTypeException;
+use Symfony\Component\Notifier\Message\MessageInterface;
+use Symfony\Component\Notifier\Message\PushMessage;
+use Symfony\Component\Notifier\Message\SentMessage;
+use Symfony\Component\Notifier\Transport\AbstractTransport;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+use Symfony\Contracts\HttpClient\Exception\TransportExceptionInterface;
+use Symfony\Contracts\HttpClient\HttpClientInterface;
+
+/**
+ * @author Mickael Perraud <mikaelkael.fr@gmail.com>
+ */
+final class NtfyTransport extends AbstractTransport
+{
+    protected const HOST = 'ntfy.sh';
+    private ?string $user = null;
+    private ?string $password = null;
+    private string $topic;
+    private bool $secureHttp = true;
```

Comment:
I am not sure if should allow http in any case, event if the provider supports it.
Is there a special use case for that?